src/main/java/org/folio/service/logs/ErrorLogServiceImpl.java

```diff
@@ -22,20 +30,53 @@
   }
 
   @Override
-  public Future<ErrorLog> save(ErrorLog errorLog, OkapiConnectionParams params) {
+  public Future<ErrorLog> save(ErrorLog errorLog, String tenantId) {
     if (errorLog.getId() == null) {
       errorLog.setId(UUID.randomUUID().toString());
     }
-    return errorLogDao.save(errorLog, params.getTenantId());
+    errorLog.setCreatedData(new Date());
```

Comment:
It should be really `setCreatedDate`. And also the field should be named `createDate`.